### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by musangamfure**
Hello, 

In this PR I have worked on the Popup Window and made the work section dynamic using JS. Kindly let me know what you think about my code.

Best Regards.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -46,3 +46,170 @@ form.addEventListener('submit', (e) => {
...
+    title: 'Tonic',
+    client: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tags: ['HTML', 'CSS', 'Ruby on Rails', 'JavaScript'],
+    image: './images/nature.png',
+    modaltext:
+      "  Lorem Ipsum is simply dummy text of the printing and typesettin industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley otype and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap intoelectronic typesetting, remaining essent",
```
Comment (Ismailco) [inline script.js]: - [ ] Greate work creating the project object however you should add a key for the `live link` and `source link` you your object list

File: index.html [hunk 2]
```diff
@@ -92,131 +92,7 @@ <h1>
...
-              <h3>JavaScript</h3>
-            </div>
-            <button class="simple-button" type="button">See Project</button>
-          </div>
-          <div class="work-col-1">
-            <img class="snapshoot" src="./images/uber.png" alt="" />
-          </div>
-        </div>
-      </section>
+      <section class="works" id="portfolio"></section>
```
Comment (Ismailco) [inline index.html:95]: - [ ] Good job creating the projects and popups dynamically, however, I can't scroll down when I open the popup, which makes it hard to navigate the popup, kindly make sure to check your styling

Screenshot_20230415_134528

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @musangamfure, @clin2on3mun  

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally
✔️ Clean code
✔️ Nice UI

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi  Team,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**